cf_xarray/accessor.py

```diff
@@ -1523,6 +1531,69 @@ def differentiate(
                 result *= -1
         return result
 
+    def add_cf_attributes(
```

Comment:
```suggestion
    def add_canonical_cf_attributes(
```

I'm 50/50 on this suggestion